packages/local-cli/core/__tests__/findAssets-test.js

```diff
@@ -13,7 +13,7 @@
 jest.mock('path');
 jest.mock('fs');
 
-const findAssets = require('../findAssets');
+const { findAssets } = require('../getAssets');
```

Comment:
yo, your prettier config should not have these whitespaces.

Reply:
Well, we don't have prettier set up in the repository just yet. I was waiting till we finish "migration" with that and then, I started this PR.

I was planning to use our own preset https://github.com/callstack/eslint-config-callstack as it already has `prettier` integration built into ESLint rules (it automatically configures itself based on ESLint settings).

That way we only need to run `eslint --fix`.

I would suggest to "temporarily" ignore styling issues (as long as they are not that "annoying") and I'll follow up with "prettier" PR right after this one is merged.